lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -213,7 +213,7 @@ def initial_excluded_gems
 
           # If the transitive dependency is included as a transitive dependency of a gem outside of the development
           # group, skip it
-          next if others.any? { |d| d.to_spec.dependencies.include?(transitive_dependency) }
+          next if others.any? { |d| d.to_spec && d.to_spec.dependencies.include?(transitive_dependency) }
```

Comment:
```suggestion
          next if others.any? { |d| d&.to_spec.dependencies.include?(transitive_dependency) }
```

Reply:
Wouldn't this cause `dependencies` to be called on `nil`?

Reply:
Yeah, sorry, I meant with safe navigation in all chained methods. I'm fine with leaving it was is too.